integration-tests/air/test_spells.inc_other_spell.air

```diff
@@ -0,0 +1,29 @@
+; This file is auto-generated. Do not edit manually: changes may be erased.
```

Comment:
I don't think you need to commit this 🤔 It should be generated from Aqua on the fly

Reply:
I guess .gitignore `air` directory will do

Reply:
I added this directory intentionally. These air files aren't in use, I copied this text to the python files directly. I left it for clarity of what goes from where.